src/authentication/__tests__/abax-auth.test.ts

```diff
@@ -96,11 +96,11 @@ describe('abax auth', async () => {
         method: 'POST',
       })
       .reply(200, {
-        access_token: 'new_access_token',
-        expires_in: 3600,
-        id_token: 'id_token',
-        token_type: 'Bearer',
-        refresh_token: 'new_refresh_token',
+        accessToken: 'new_access_token',
+        expiresIn: 3600,
+        idToken: 'id_token',
+        tokenType: 'Bearer',
+        refreshToken: 'new_refresh_token',
```

Comment:
This change should be reverted. This is an intercept/mock of the actual Abax API, which does reply with the data in this shape.